src/chains/ethereum/transaction/src/eip2930-access-list-transaction.ts

```diff
@@ -162,6 +158,9 @@ export class EIP2930AccessListTransaction extends RuntimeTransaction {
       }
     };
   }
+  public calculateIntrinsicGas(): bigint {
+    return super.calculateIntrinsicGas() + this.accessListDataFee;
```

Comment:
This is a bug fix. We weren't include these costs in intrinsic costs when we should.